test/blackbox/common/mock/BlackboxMockConsumer.h

```diff
@@ -42,6 +42,12 @@ class BlackboxMockConsumer : public LogConsumer
         return mEntriesConsumed;
     }
 
+    void ClearEntries()
+    {
+        std::unique_lock<std::mutex> guard(mMutex);
+        mEntriesConsumed.clear();
```

Comment:
We are modifying the entries, so we should notify the condition.

```suggestion
        mEntriesConsumed.clear();
        cv_.notify_all();
```